pyramid_oereb/views/webservice.py

```diff
@@ -452,14 +466,14 @@ def __parse_gnss__(self, gnss):
 
 
 class Parameter(object):
-    def __init__(self, flavour, format, with_geometry, images, identdn=None, number=None,
-                 egrid=None, language=None, topics=None):
+    def __init__(self, response_format, flavour=None, with_geometry=False, images=False, identdn=None,
```

Comment:
Why change the order of parameters?

Reply:
The `Parameter` object is now passed to the readers and sources. Some of them are also used for other requests, like `capabilities`, that only have the format parameter. So I had to turn some of the arguments into optional keyword arguments and therefore move them after the mandatory arguments.

I also renamed the format argument as `format` is a built-in function.

Reply:
Ah I see, thanks for the explanation